test/getDeployedEntitiesStream.spec.ts

```diff
@@ -18,11 +19,13 @@ test('getDeployedEntitiesStream', ({ components, stubComponents }) => {
   ]
   it('prepares the endpoints', () => {
     // serve the snapshots
-    components.router.get('/snapshot', async () => ({
-      body: {
+    components.router.get('/snapshots', async () => ({
```

Comment:
maybe we should leave a test for the old flow?

Reply:
Yes, there are